app/src/main/java/com/bigyoshi/qrhunt/qr/FragmentScanner.java

```diff
@@ -72,28 +76,52 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         codeScanner = new CodeScanner(activity, scannerView);
 
         codeScanner.setCamera(CodeScanner.CAMERA_BACK);
-        codeScanner.setScanMode(ScanMode.PREVIEW);
+        codeScanner.setScanMode(ScanMode.CONTINUOUS);
         codeScanner.setAutoFocusMode(AutoFocusMode.SAFE);
         codeScanner.setFlashEnabled(false);
         codeScanner.setAutoFocusEnabled(true);
         codeScanner.setFormats(CodeScanner.ALL_FORMATS);
 
+        /*button = root.findViewById(R.id.start_scan);
+        button.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                codeScanner.setScanMode(ScanMode.PREVIEW);
+                camera.processQRCode();
+                button.setVisibility(View.GONE);
+                codeScanner.setScanMode(ScanMode.CONTINUOUS);
+            }
+        });*/
+
```

Comment:
no point leaving this dead code here unless it's like...useful

Reply:
This was for if we ever want to go back to button than auto scan